packages/ui5-application-writer/src/types.ts

```diff
@@ -63,6 +63,7 @@ export interface UI5 {
     typesPackage: string;
     descriptorVersion: string;
     ui5Libs: string | string[];
+    manifestLibs?: string | string[];
```

Comment:
@martinjanus-sap Can you please document the changes and the behaviour for consumers of the API please? Adding tsdocs here should suffice but please also the `ui5Libs` property.

Reply:
Please check commit - https://github.com/SAP/open-ux-tools/pull/2280/commits/d65f0bd376fa569dd30b78456abcacbb882d6664 

For some types I tried my best to infer usage and give appropriate doc